internal/ctxval/context_values.go

```diff
@@ -2,14 +2,22 @@ package ctxval
 
 import (
 	"context"
+	"encoding/base64"
+	"encoding/json"
 	"errors"
+	"fmt"
 
 	ucontext "github.com/Unleash/unleash-client-go/v3/context"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 )
 
+var (
+	DecodeErr           = fmt.Errorf("unable to b64 decode x-rh-identity header")
+	MissingValidJSONErr = fmt.Errorf("x-rh-identity header does not contain valid JSON")
```

Comment:
Use `errors.New` here this is a dynamic error assignment, our linter should complain.